rest/route/project.go

```diff
@@ -748,3 +750,76 @@ func (h *getProjectVersionsHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return resp
 }
+
+type testProjectAliasHandler struct {
+	sc data.Connector
+
+	version             string
+	alias               string
+	includeDependencies bool
+	project             *dbModel.Project
+}
+
+func makeTestProjectAliasHandler(sc data.Connector) gimlet.RouteHandler {
+	return &testProjectAliasHandler{
+		sc: sc,
+	}
+}
+
+func (h *testProjectAliasHandler) Factory() gimlet.RouteHandler {
+	return &testProjectAliasHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *testProjectAliasHandler) Parse(ctx context.Context, r *http.Request) error {
+	params := r.URL.Query()
+
+	h.version = params.Get("version")
```

Comment:
Why is version  a requirement? Could a user just give a project and use the current updated yaml instead of having to pass in a version?

Reply:
This is mostly to satisfy cloud's use case. They need to be able to do this for patches, which can be done with sending a config in the body, but I don't want them to have to roundtrip that